components/drafts/draft_title/draft_title.tsx

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {memo, useEffect} from 'react';
+import {useDispatch} from 'react-redux';
+import {FormattedMessage} from 'react-intl';
+
+import {getMissingProfilesByIds} from 'mattermost-redux/actions/users';
+import {Channel} from 'mattermost-redux/types/channels';
+import {UserProfile} from 'mattermost-redux/types/users';
+
+import {imageURLForUser} from 'utils/utils.jsx';
+import {Constants} from 'utils/constants';
+
+import Avatar from 'components/widgets/users/avatar';
+
+import './draft_title.scss';
+
+type Props = {
+    channel: Channel;
```

Comment:
There seems that you only need the channel type, no need for the whole channel.

Reply:
My previous comment here is still valid. In this component you don't need the whole channel, just the type of channel.